server/routes/users.js

```diff
@@ -84,24 +80,6 @@ async function logUser(req, res) {
   }
 }
 
```

Comment:
@AStashevskaya Received this error after creating new user

<img width="539" alt="image" src="https://user-images.githubusercontent.com/12561302/120528953-48691600-c3dc-11eb-8e39-ae2962552e01.png">